api/codelists.go

```diff
@@ -32,10 +53,17 @@ func (c *CodeListAPI) getCodeLists(w http.ResponseWriter, r *http.Request) {
 		codeLists.Items[i] = item
 	}
 
-	count := len(codeLists.Items)
+	sort.Slice(codeLists.Items, func(i, j int) bool {
```

Comment:
Has there been some discussion whether to try and do the pagination and sorting in the database? It's not ideal to do it in code ourselves, as the database will still have to return the full set of data. That being said, I am not sure how practical it is to do it in the graph database.